packages/flipper-plugin-react-navigation/src/Logs.tsx

```diff
@@ -125,7 +123,6 @@ const EmptyIcon = styled(CompassOutlined)({
 });
 
 const BlankslateText = styled.h5({
-  color: 'rgba(0, 0, 0, 0.85)',
```

Comment:
This is to make the text appear lower opacity. If you're removing the color, it needs to add other styling like opacity to keep the same effect.